### DISCUSSION TITLE
Infuse

### DISCUSSION THREAD

**Discussion by berot3** [midarrlabs/midarr-server#391] [Ideas]
Hi! To be able to use Midarr on a Apple TV, it would be nice if there were support for Midarr in Infuse. Infuse already supports emby, jellyfin and plex. Maybe you could reach out to infuse?

**Comment by trueChazza** [comment]
Ah interesting! Thanks for opening this.
This would require Midarr APIs to be fleshed out abit more (which I'm currently doing). It wouldn't have to be completely ready - just having the interface / contract in place would be enough I think to take to Infuse.

Great idea though!
